scripts/ci/dockerfiles/stress/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG ALPINE_VERSION="3.12"
+
+FROM alpine:${ALPINE_VERSION}
+
+ARG STRESS_VERSION
+ARG AIRFLOW_STRESS_VERSION
+ARG COMMIT_SHA
+
+MAINTAINER "Apache Airflow Community <dev@airflow.apache.org>"
```

Comment:
According to Official Docker Documentation the MAINTAINER instruction is deprecated. Instead, one should use the LABEL instruction to define the author of the generated images. The LABEL instruction is more flexible, enables setting metadata, and can be easily viewed with the command docker inspect .


Same here

Reply:
Yep. I am goint to rebase and apply the hadolint rules from master.